app.py

```diff
@@ -6,8 +6,10 @@
 from dotenv import load_dotenv
 from quart import Quart, request, jsonify
 from quart_cors import cors
+import time
 import azure_storage.azure_storage_api as azure_storage_api
 import model_inference.inference as inference
+import model_utilitary_functions.model_UTILS as utils
```

Comment:
https://breadcrumbscollector.tech/stop-naming-your-python-modules-utils/

Reply:
Change it to model_request and move the function that processes data return from the models to the inference module.

Add the following workflows jobs:
* repo-standards-validation
* markdown and YAML check

Got the following failed checks after adding them: 
* No TESTING.md from repo-standard-validation
* Various Markdown errors from the markdown check
* No workflows.yml file found from the YAML check

The next commit will correct the failed checks.